pages/about_data_page.py

```diff
@@ -1,12 +1,242 @@
+                | Baseline            | 3.0  | Zero from 2028 | 1%                        | Yes                    | 5%                 | Reference                                                 | No rebalancing                                |
+                | High innovation     | 3.5  | Fast stepdown  | 5%                        | Yes                    | 5%                 | Reference                                                 | Rebalance RO and FiT from electricity to gas  |
+                | Cheaper electricity | 3.0  | Fast stepdown  | 1%                        | Yes                    | 5%                 | Reference for gas, Low fossil fuel prices for electricity | Rebalance RO and FiT from electricity to gas  |
+                | High subsidy        | 3.5  | High           | 1%                        | Yes                    | 5%                 | Reference                                                 | Rebalance RO and FiT from electricity to gas  |
+                
+                In the 'scenario selection' page, the user can select one of the pre-set scenarios or create a custom scenario by changing any of the parameters.
+                The 'comparing scenarios' page compares lifetime costs of gas boilers and air source heat pumps under the above pre-set scenarios.
+
+                Below you can find more information about the methodology and data sources used in the lifetime costs calculator.
+                """
+    )
+    
+    st.markdown("## Methodology")
+
+    st.markdown(f"""Methodology for calculating lifetime costs of heating systems. 
+                Parameters highlighted in <span style='color:{nesta_blue};'>blue</span> are changeable inputs by the user in the app.""", unsafe_allow_html=True)
+
+    with st.expander("Total lifetime costs"):
+        # Lifetime cost calculations
+        st.markdown(f"""
+                    **Total lifetime cost**: This is the total cost of owning a heating system over its entire lifetime.
+                    
+                    Total lifetime cost [£] = Upfront costs [£] + Maintenance costs over lifetime [£] + Running costs over lifetime [£]
+
+                    **Annualised lifetime cost**: This is the lifetime cost averaged over the heating system's lifetime.
+
+                    Annualised lifetime cost [£/year] = Total lifetime cost [£] / <span style='color:{nesta_blue};'>Lifespan</span> [years]
+
+                    **Time series breakdown of total cost over the heating system's lifetime**: This is the total cost of having that heating system in each year of its lifetime. This cost will be different for each year of its lifetime.
+
+                    For each year of the heating system's lifetime:
+
+                    Total cost of owning the heating system [£/year] = Upfront cost [£/year] + Maintenance cost [£/year] + Running cost [£/year]
+                    """)
```

Comment:
```suggestion
                    """, unsafe_allow_html=True)
```

The text that is supposed to be blue isn't showing blue on the app - I think it's because you're missing this additional argument